ctl/control_tsmp2.sh

```diff
@@ -92,10 +92,10 @@ echo "geo_dir: "${geo_dir}
 # select machine defaults, if not set by user
 if ( [ -z $npnode_u ] | [ -z $partition_u ] ); then
 echo "Take system default for npnode and partition. "
-if [ ${SYSTEMNAME^^} == "JUWELS" ];then
+if [ "${SYSTEMNAME}" == "juwels" ]; then
```

Comment:
Even though I prefer the lowercase writing, I would stick to the uppercase case writing for consistency reasons.